web/src/components/users/FirstUser.test.jsx

```diff
@@ -41,6 +41,16 @@ let setUserFn = jest.fn().mockResolvedValue(setUserResult);
 const removeUserFn = jest.fn();
 let onUsersChangeFn = jest.fn();
 
+const openUserForm = async () => {
+  const { user } = installerRender(<FirstUser />);
+  await screen.findByText("No user defined yet.");
+  const button = await screen.findByRole("button", { name: "Define a user now" });
+  await user.click(button);
+  const dialog = await screen.findByRole("dialog");
+
+  return { user, dialog }
+}
```

Comment:
NP: Missing semicolons, 

```suggestion
  return { user, dialog };
};
```